src/composables/queries/useGroupsQuery.js

```diff
@@ -0,0 +1,39 @@
+import { computed, toValue } from 'vue';
+import { useQuery } from '@tanstack/vue-query';
+import _isEmpty from 'lodash/isEmpty';
+import { orgFetcher } from '@/helpers/query/orgs';
+import useUserType from '@/composables/useUserType';
+import useUserClaimsQuery from '@/composables/queries/useUserClaimsQuery';
+import { GROUPS_QUERY_KEY } from '@/constants/queryKeys';
+import { FIRESTORE_COLLECTIONS } from '@/constants/firebase';
+
+/**
+ * Groups query.
+ *
+ * @param {Ref<String>} districtId – A Vue ref containing the ID of the district to fetch schools for.
```

Comment:
Is this left over from a copy/paste? Groups should not require a district as it's an independent orgType. Perhaps this needs to be groupId.

Reply:
Whoops, good catch – removed, thank you!